packages/teleport/src/components/ResourceButtonAdd/index.ts

```diff
@@ -0,0 +1,3 @@
+import ResourceButtonAdd from './ResourceButtonAdd';
```

Comment:
i think we should start using `agent` to mean `node,app,db,kube,desktop`, so `AgentButtonAdd`? [Resource](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/services/resources/resource.ts) kinda means something else in the front (which is everything else other than agents which is auth connectors, roles, trusted cluster). I think zac says we often refer to node,app,kube,desktop,db as agents also.